include/deal.II/base/quadrature_lib.h

```diff
@@ -746,6 +746,34 @@ class  QTrianglePolar: public QSimplex<2>
   QTrianglePolar(const unsigned int &n);
 };
 
+
+/**
+ * A two dimensional simplex quadrature with zero Jacobian on vertex zero. The
+ * quadrature is obtained through the so called Duffy transformation that maps
+ * a standard Gaussian quadrature on the unit square to the Simplex Triangle:
+ *
+ * \f[
+ *  \begin{pmatrix}
+ *  x \\
+ *  y
+ *  \end{pmatrix}
+ *  =
+ * \begin{pmatrix}
+ *  \hat{x}^\beta * (1-v) \\
+ *  \hat{x}^\beta * v
+ *  \end{pmatrix}
+ * \f]
+ *
+ * @author Nicola Giuliani, 2017
+ */
+
+class  QDuffy: public QSimplex<2>
+{
+public:
+
+  QDuffy(const unsigned int &n, const unsigned int &beta=1.);
```

Comment:
Can you add another constructor, like the other two quadratures, that uses different quadrature formulas in the x and y direction? 

Then this would just call the other one with QGauss<1>(n), QGauss<1>(n).

Just like LachatWatson, and TrianglePolar